bin/metadata_validation.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+import json
+
+import synapseclient
+from synapseclient import Entity
+
+
+def get_synaspe_file(syn_id):
+    """
+    Initializes synapseclient session and retrieves file metadata
+    """
+    syn = synapseclient.Synapse()
+    syn.login(authToken=os.environ["SYNAPSE_AUTH_TOKEN"])
```

Comment:
Nit: you technically don't need to do this, because the Python client actually directly reads from the `SYNAPSE_AUTH_TOKEN` env variable. 

You can also do the follow as a shortcut:
```
syn = synapseclient.login()
```